src/components/daydream-controls.js

```diff
@@ -18,14 +16,16 @@ var GAMEPAD_ID_PREFIX = 'Daydream Controller';
  */
 module.exports.Component = registerComponent('daydream-controls', {
   schema: {
-    hand: {default: DEFAULT_HANDEDNESS}, // This informs the degenerate arm model.
+    hand: {default: ''}, // This informs the degenerate arm model.
```

Comment:
it doesn't matter how mny are specified s long as only one is matched, and even then i don't agree in all cases that only one should be allowed, since you may have more than one object that should be controlled by the daydream controller, just like oculus or vive or soon gearvr.